ccm_web/server/src/api/api.course.handler.ts

```diff
@@ -107,7 +107,9 @@ export class CourseApiHandler {
 
   async createSections (sections: string[]): Promise<CanvasCourseSection[] | APIErrorData> {
     const start = process.hrtime()
-    const apiPromises = sections.map(async (section) => await this.createSection(section))
+    const apiPromises = createLimitedPromises<CanvasCourseSection | APIErrorData>(
+      sections.map(section => async () => await this.createSection(section))
+    )
```

Comment:
Probably okay without this limiting here also, since the limit is 60, but it won't hurt anything either.

Reply:
Sure, Now that I think about it. Doesn't hurt to have it on create section as well